Add keyboard shortcuts to focus search field

aadnekar opened the pull request:
For now the supported shortcuts will be `ctrl + f`, `ctrl + e`,
and `ctrl + l` where all of them will focus the search field (do
exactly the same).

nicolai-nordic commented on src/launcher/components/AppManagementFilter.jsx:
```diff
@@ -26,6 +26,25 @@ export const sortedSources = sources => {
+        document.addEventListener(
+            'keydown',
+            event => {
+                const keyName = event.key;
+                if (event.ctrlKey) {
+                    if (keyName === 'e' || keyName === 'f' || keyName === 'l') {
```
Very minor suggested change:

```suggestion
                const { key } = event;
                if (event.ctrlKey) {
                    if (['e', 'f', 'l'].includes(key)) {
```

aadnekar commented:
> We have a `useHotKey` hook in shared to define keyboard shortcuts. Please have a look whether you can use that.
> 
> Are the shortcuts on macOS (also) Cmd-e/f/l? I assume on a Mac this would be more fitting, unless Cmd-… is already used for something more important.

I've looked into it and implemented the same functionality using `useHotKey`, which seem very clean. Added the command+<key> also. Thanks for pointing it out, Marko :)

aadnekar commented on Changelog.md:
```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 ### Added
 - Search and filter header is now sticky when scrolling
+- Keyboard shortcuts to focus search field: `ctrl + e`, `ctrl + f`, `ctrl + l`.
```
```suggestion
- Keyboard shortcuts to focus search field: `ctrl + e` or `cmd + e`
```

datenreisender reviewed (approved):
Made a really small change to the Changelog. 

I suggest that you merge this as it is. But afterwards please have a look at one more thing and see whether you can implement it a bit better:

In https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/ae5c88f74cee4ae85138346395dedcee9b8f463e/src/launcher/components/AppManagementFilter.jsx#L142 you depend on a classname and DOM element. If this changes (e.g. because we decide to use CSS modules or Form.Control is reimplemented using a different DOM element) this would break. I think the more “React way”™  of doing it would be to [attach a ref to the input element](https://reactjs.org/docs/refs-and-the-dom.html) and use that to focus the element. The second best way would be to define an id on the input element and use that. But first have a look whether you can make it work using a ref.
